drivers/power/pm/Kconfig

```diff
@@ -77,6 +87,43 @@ config PM_GOVERNOR_EXPLICIT_RELAX
 		if set to timeout (unit: ms), that means pm_staytimeout(ms).
 		pm_relax() will be auto called after timeout.
 
+if PM_GOVERNOR_STABILITY
+
+config PM_GOVERNOR_STABILITY_IDLE_THRESH
+	int "Enter idle thresh >= (ticks)"
+	default 0
+	---help---
+		only if first time try goto idle, can remain wfi for (ticks)
+		and second time still try goto this state, goto idle,
+		otherwise keep last state
+
+		only if remained for >=(ticks), allow goto idle state
+		set to 0 disable stability check for idle state
+
+config PM_GOVERNOR_STABILITY_STANDBY_THRESH
+	int "Enter standby thresh >= (ticks)"
+	default 0
+	---help---
+		only if first time try goto standby, can remain wfi for (ticks)
+		and second time still try goto this state, goto standby,
+		otherwise keep last state
+
+		only if remained for >=(ticks), allow goto standby state
```

Comment:
Please fix this broken line

Reply:
add the missing periods will be OK? or should remove the empty line.